### ISSUE TITLE
NVDA does not read the items within file explorer for the first time

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#5817]
NVDA does not read the items inside this computer.
 To reproduce the error:
 Open the start menu.
 walk with the arrows below to file explorer, press enter.
 Expected, NVDA will read the items inside this computer.
 What happens, NVDA does not read the names of folders and drives.
 This problem occurs intermittently often NVDA reads items normally.
 The following is a log showing the errors that occur shortly after getting into the file explorer:

Input: kb(laptop):e
IO - speech._speakSpellingGen (20:31:55):
Speaking character u'e'
IO - speech.speak (20:31:55):
Speaking [u'desconhecido']
DEBUGWARNING - watchdog._watcher (20:31:56):
Trying to recover from freeze, core stack:
  File "nvda.pyw", line 191, in <module>
  File "core.pyc", line 382, in main
  File "wx_core.pyc", line 8657, in MainLoop
  File "wx_core.pyc", line 7952, in MainLoop
  File "core.pyc", line 352, in Notify
  File "IAccessibleHandler.pyc", line 884, in pumpAll
  File "IAccessibleHandler.pyc", line 617, in processGenericWinEvent
  File "IAccessibleHandler.pyc", line 529, in winEventToNVDAEvent
  File "NVDAObjects\IAccessible__init__.pyc", line 43, in getNVDAObjectFromEvent
  File "NVDAObjects__init__.pyc", line 69, in **call**
  File "NVDAObjects\IAccessible__init__.pyc", line 554, in **init**
  File "IAccessibleHandler.pyc", line 904, in getIAccIdentity
  File "comtypes__init__.pyc", line 1078, in QueryInterface

DEBUGWARNING - core.Notify (20:31:56):
errors in this core pump cycle
Traceback (most recent call last):
  File "core.pyc", line 352, in Notify
  File "IAccessibleHandler.pyc", line 884, in pumpAll
  File "IAccessibleHandler.pyc", line 617, in processGenericWinEvent
  File "IAccessibleHandler.pyc", line 529, in winEventToNVDAEvent
  File "NVDAObjects\IAccessible__init__.pyc", line 43, in getNVDAObjectFromEvent
  File "NVDAObjects__init__.pyc", line 69, in **call**
  File "watchdog.pyc", line 198, in _COMError_init
CallCancelled
DEBUGWARNING - watchdog._watcher (20:31:57):
Trying to recover from freeze, core stack:
  File "nvda.pyw", line 191, in <module>
  File "core.pyc", line 382, in main
  File "wx_core.pyc", line 8657, in MainLoop
  File "wx_core.pyc", line 7952, in MainLoop
  File "core.pyc", line 352, in Notify
  File "IAccessibleHandler.pyc", line 884, in pumpAll
  File "IAccessibleHandler.pyc", line 617, in processGenericWinEvent
  File "IAccessibleHandler.pyc", line 521, in winEventToNVDAEvent
  File "_UIAHandler.pyc", line 279, in isUIAWindow
  File "_UIAHandler.pyc", line 273, in _isUIAWindowHelper

DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:57):
oleacc.AccessibleObjectFromEvent with window 132304, objectID 100728883 and childID 0: [Error -2147467259] Erro não especificado
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:57):
oleacc.AccessibleObjectFromEvent with window 132304, objectID 100728885 and childID 0: [Error -2147467259] Erro não especificado
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:57):
oleacc.AccessibleObjectFromEvent with window 132304, objectID 100728886 and childID 0: [Error -2147467259] Erro não especificado
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:57):
oleacc.AccessibleObjectFromEvent with window 132304, objectID 100794393 and childID 0: [Error -2147467259] Erro não especificado
DEBUGWARNING - watchdog._watcher (20:31:57):
Trying to recover from freeze, core stack:
  File "nvda.pyw", line 191, in <module>
  File "core.pyc", line 382, in main
  File "wx_core.pyc", line 8657, in MainLoop
  File "wx_core.pyc", line 7952, in MainLoop
  File "core.pyc", line 353, in Notify
  File "queueHandler.pyc", line 83, in pumpAll
  File "queueHandler.pyc", line 47, in flushQueue
  File "comtypesMonkeyPatches.pyc", line 54, in newCpbDel
  File "comtypes__init__.pyc", line 841, in **del**
  File "comtypes__init__.pyc", line 1092, in Release

DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:58):
oleacc.AccessibleObjectFromEvent with window 132334, objectID 245 and childID 0: [Error -2147467259] Erro não especificado
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:58):
oleacc.AccessibleObjectFromEvent with window 132334, objectID 248 and childID 0: [Error -2147467259] Erro não especificado
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:58):
oleacc.AccessibleObjectFromEvent with window 132334, objectID 249 and childID 0: [Error -2147467259] Erro não especificado
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:58):
oleacc.AccessibleObjectFromEvent with window 132334, objectID 252 and childID 0: [Error -2147467259] Erro não especificado
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:58):
oleacc.AccessibleObjectFromEvent with window 132334, objectID 253 and childID 0: [Error -2147467259] Erro não especificado

**Comment by fernando-jose-silva** [comment]
This problem also occurs when you open the Save As window.

**Comment by derekriemer** [comment]
When this occurs, do you often get a bug where NVDA cannot detect that 
the start screen appeared?

On 3/16/2016 5:49 PM, fernando-jose-silva wrote:
>
> NVDA does not read the items inside this computer.
> To reproduce the error:
> Open the start menu.
> walk with the arrows below to file explorer, press enter.
> Expected, NVDA will read the items inside this computer.
> What happens, NVDA does not read the names of folders and drives.
> This problem occurs intermittently often NVDA reads items normally.
> The following is a log showing the errors that occur shortly after 
> getting into the file explorer:
>
> Input: kb(laptop):e
> IO - speech./speakSpellingGen (20:31:55):
> Speaking character u'e'
> IO - speech.speak (20:31:55):
> Speaking [u'desconhecido']
> DEBUGWARNING - watchdog./watcher (20:31:56):
> Trying to recover from freeze, core stack:
> File "nvda.pyw", line 191, in
> File "core.pyc", line 382, in main
> File "wx_core.pyc", line 8657, in MainLoop
> File "wx_core.pyc", line 7952, in MainLoop
> File "core.pyc", line 352, in Notify
> File "IAccessibleHandler.pyc", line 884, in pumpAll
> File "IAccessibleHandler.pyc", line 617, in processGenericWinEvent
> File "IAccessibleHandler.pyc", line 529, in winEventToNVDAEvent
> File "NVDAObjects\IAccessible__init//.pyc", line 43, in 
> getNVDAObjectFromEvent
> File "NVDAObjects__init__.pyc", line 69, in *call*
> File "NVDAObjects\IAccessible__init__.pyc", line 554, in *init*
> File "IAccessibleHandler.pyc", line 904, in getIAccIdentity
> File "comtypes__init__.pyc", line 1078, in QueryInterface
>
> DEBUGWARNING - core.Notify (20:31:56):
> errors in this core pump cycle
> Traceback (most recent call last):
> File "core.pyc", line 352, in Notify
> File "IAccessibleHandler.pyc", line 884, in pumpAll
> File "IAccessibleHandler.pyc", line 617, in processGenericWinEvent
> File "IAccessibleHandler.pyc", line 529, in winEventToNVDAEvent
> File "NVDAObjects\IAccessible__init__.pyc", line 43, in 
> getNVDAObjectFromEvent
> File "NVDAObjects__init__.pyc", line 69, in *call*
> File "watchdog.pyc", line 198, in _COMError_init
> CallCancelled
> DEBUGWARNING - watchdog._watcher (20:31:57):
> Trying to recover from freeze, core stack:
> File "nvda.pyw", line 191, in
> File "core.pyc", line 382, in main
> File "wx_core.pyc", line 8657, in MainLoop
> File "wx_core.pyc", line 7952, in MainLoop
> File "core.pyc", line 352, in Notify
> File "IAccessibleHandler.pyc", line 884, in pumpAll
> File "IAccessibleHandler.pyc", line 617, in processGenericWinEvent
> File "IAccessibleHandler.pyc", line 521, in winEventToNVDAEvent
> File "_UIAHandler.pyc", line 279, in isUIAWindow
> File "_UIAHandler.pyc", line 273, in _isUIAWindowHelper
>
> DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:57):
> oleacc.AccessibleObjectFromEvent with window 132304, objectID 
> 100728883 and childID 0: [Error -2147467259] Erro não especificado
> DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:57):
> oleacc.AccessibleObjectFromEvent with window 132304, objectID 
> 100728885 and childID 0: [Error -2147467259] Erro não especificado
> DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:57):
> oleacc.AccessibleObjectFromEvent with window 132304, objectID 
> 100728886 and childID 0: [Error -2147467259] Erro não especificado
> DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:57):
> oleacc.AccessibleObjectFromEvent with window 132304, objectID 
> 100794393 and childID 0: [Error -2147467259] Erro não especificado
> DEBUGWARNING - watchdog./watcher (20:31:57):
> Trying to recover from freeze, core stack:
> File "nvda.pyw", line 191, in
> File "core.pyc", line 382, in main
> File "wx_core.pyc", line 8657, in MainLoop
> File "wx_core.pyc", line 7952, in MainLoop
> File "core.pyc", line 353, in Notify
> File "queueHandler.pyc", line 83, in pumpAll
> File "queueHandler.pyc", line 47, in flushQueue
> File "comtypesMonkeyPatches.pyc", line 54, in newCpbDel
> File "comtypes__init_/.pyc", line 841, in *del*
> File "comtypes__init__.pyc", line 1092, in Release
>
> DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:58):
> oleacc.AccessibleObjectFromEvent with window 132334, objectID 245 and 
> childID 0: [Error -2147467259] Erro não especificado
> DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:58):
> oleacc.AccessibleObjectFromEvent with window 132334, objectID 248 and 
> childID 0: [Error -2147467259] Erro não especificado
> DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:58):
> oleacc.AccessibleObjectFromEvent with window 132334, objectID 249 and 
> childID 0: [Error -2147467259] Erro não especificado
> DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:58):
> oleacc.AccessibleObjectFromEvent with window 132334, objectID 252 and 
> childID 0: [Error -2147467259] Erro não especificado
> DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (20:31:58):
> oleacc.AccessibleObjectFromEvent with window 132334, objectID 253 and 
> childID 0: [Error -2147467259] Erro não especificado
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/nvaccess/nvda/issues/5817>
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194

**Comment by fernando-jose-silva** [comment]
Good afternoon.
At Least on my machine, not the start menu of Windows 10 is always to be read.
Thank you very much.

**Comment by ehollig** [comment]
@fernando-jose-silva, is this still an issue?

**Comment by Adriani90** [comment]
this might be related to #7237.

**Comment by Adriani90** [comment]
and also to #7982 including referenced issues.
Please note the comments provided by @dineshkaushal.

**Comment by fernando-jose-silva** [comment]
the problem does not happen anymore, thank you very much.

Enviado do meu iPhone

Em 11 de mar de 2018, à(s) 11:56, Adriani90 <notifications@github.com> escreveu:

**Comment by ehollig** [comment]
Closing per above comment.
